plugins/glustershd/actor.go

```diff
@@ -13,75 +14,149 @@ import (
 
 var names = [...]string{"replicate", "afr"}
 
+var selfhealKeys = []string{"replicate.self-heal-daemon", "afr.self-heal-daemon"}
+
 const (
 	selfHealKey = "self-heal-daemon"
 )
 
 type shdActor struct{}
 
-func (actor *shdActor) Do(v *volume.Volinfo, key string, value string, logger log.FieldLogger) error {
-	if key != selfHealKey {
+func (actor *shdActor) Do(v *volume.Volinfo, key string, value string, volOp xlator.VolumeOpType, logger log.FieldLogger) error {
```

Comment:
@vpandey-RH Can you review this file ?

Reply:
@prashanthpai I have tested the glustershd behaviour with this patch. Seems to work fine. 
Cannot comment on the code as I am not very aware of the code architecture being followed here.